File: 2624-DifferenceBetweenElementSumAndDigitSumOfAnArray/2624-DifferenceBetweenElementSumAndDigitSumOfAnArray.js
// Last updated: 3/6/2026, 2:08:34 PM
/**
 * @param {number[]} nums
 * @return {number}
 */
var differenceOfSum = function(nums) {
    let elements = 0;
    let digit=0;
    for(let i=0;i<nums.length;i++){
        elements=elements+nums[i];
        let num = nums[i]
        while(num>0){
            digit += num % 10;
            num = Math.floor(num / 10);
        }
    }
      return Math.abs(elements - digit);
};
console.log(differenceOfSum([1,15,6,3]));

File: 2624-DifferenceBetweenElementSumAndDigitSumOfAnArray/2624-DifferenceBetweenElementSumAndDigitSumOfAnArray.cpp
// Last updated: 7/17/2026, 4:24:32 PM
class Solution {
public:
    int differenceOfSum(vector<int>& nums) {
        int elements = 0;
        int digit = 0;

        for (int i = 0; i < nums.size(); i++) {
            elements += nums[i];

            int num = nums[i];
            while (num > 0) {
                digit += num % 10;
                num /= 10;
            }
        }

        return abs(elements - digit);
    }
};